public/data/bala-hantsi.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "Bala Hantsi",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/bhantsi.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/bhantsi",
+      "icon": "github"
+    },
+    {
+      "name": "Bala Hantsi",
+      "url": "https://twitter.com/b_hantsi",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "December 2018",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing"
+    },
+    {
+      "title": "Started Mentoring",
+      "date": "March 2019",
+      "description": "Started mentoring"
+    },
+    {
+      "title": "Started engaigin in developers community",
```

Spelling error here; might want to fix. 🙂
```suggestion
      "title": "Started engaging in developers' community",
```